test/compilable/test9029.d

```
@@ -0,0 +1,17 @@
+enum NameOf(alias S) = S.stringof;
```

All test files should have a `TEST_OUTPUT` section (even if empty) to avoid that deprecations or warnings are silently ignored.

---

fixed